common/table.js

```diff
@@ -2255,6 +2255,12 @@
                     );
                 }
 
+                scope.showAddRecord = function () {
```

Comment:
I think you were meant to use this for the `ng-if` of the "create" button in `tableHeader.html`, but it's not used any where. Instead it seems like you're just doing it with css for unlink popup. 

So I think you should just remove the CSS rule and make sure this function is used in `tableHeader.html`